docs/docsite/rst/reference_appendices/module_utils.rst

```diff
@@ -25,3 +25,50 @@ To use this functionality, include ``import ansible.module_utils.basic`` in your
 
 .. automodule:: ansible.module_utils.basic
    :members:
+
+
+Argument Spec
+---------------------
+
+Classes and functions for validating parameters against an argument spec.
+
+ArgumentSpecValidator
+=====================
+
+.. autoclass:: ansible.module_utils.common.arg_spec.ArgumentSpecValidator
```

Comment:
Side note: It's possible to add `.. currentmodule:: ansible.module_utils` at the top and avoid prefixing it in every definition.